src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
It'd probably be cleaner to put rtl styles in their own file (either all RTL in one file, or like accounts-rtl.style) rather than sprinkling it over the ltr styles